pact4s/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/provider/DiskStateManager.scala

```diff
@@ -0,0 +1,135 @@
+package org.broadinstitute.dsde.workbench.leonardo.provider
+
+import cats.effect.IO
+import cats.mtl.Ask
+import org.broadinstitute.dsde.workbench.google2.{DiskName, ZoneName}
+import org.broadinstitute.dsde.workbench.leonardo.http.{
+  GetPersistentDiskResponse,
+  ListPersistentDiskResponse,
+  UpdateDiskRequest
+}
+import org.broadinstitute.dsde.workbench.leonardo.http.service.{DiskNotFoundException, DiskService}
+import org.broadinstitute.dsde.workbench.leonardo.{
+  AppContext,
+  AuditInfo,
+  BlockSize,
+  CloudContext,
+  DiskId,
+  DiskSize,
+  DiskStatus,
+  DiskType,
+  FormattedBy,
+  LabelMap,
+  SamResourceId
+}
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import org.broadinstitute.dsde.workbench.model.{TraceId, UserInfo, WorkbenchEmail}
+import org.mockito.ArgumentMatchers.{any, anyString}
+import org.mockito.Mockito.when
+import org.mockito.stubbing.OngoingStubbing
+import pact4s.provider._
+
+import java.time.Instant
+
+object DiskStateManager {
+  object States {
+    final val DiskExists = "there is a disk in a Google project"
+    final val DiskDoesNotExist = "there is not a disk in a Google project"
+    final val GoogleProjectExists = "there is a Google project with disks"
```

Comment:
do this state (`GoogleProjectExists`) need to lineup by key / definition with what is above in `AppStateManager` or anywhere else?

Reply:
No, they do not need to match key or value.

I renamed the variables so that lack of relationship is clearer.